api/location/models.py

```diff
@@ -29,6 +29,12 @@ class Location(Base, Utility):
     time_zone = Column(Enum(TimeZoneType))
     image_url = Column(String)
     state = Column(Enum(StateType), default="active")
+    structure_id = Column(
+        Integer,
+        ForeignKey('structure.id', ondelete="CASCADE"),
+        nullable=False
```

Comment:
Hi @ThaDeveloper , you need to sync with @RachaelNantale, about this line to understand why this field has `nullable= True`. She has an explanation for why according to the image below she used True instead of false in her implementation. And she can explain this better.
<img width="433" alt="screen shot 2019-03-06 at 14 54 02" src="https://user-images.githubusercontent.com/39129473/53879709-2d9e7000-4020-11e9-8d79-47776d5fd6c1.png">

Reply:
@nakatuddesuzan I had a sync with her yesterday and the concern was that introducing a non-nullable column would cause problems with the already existing database that has data, but since `v2` will use a new empty database having the `structure_id` *nullable=False* won't be an issue.